doc/content/gateways/thethingsindoorgateway/_index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "The Things Indoor Gateway"
+description: ""
+distributions: ["Community", "Cloud"]
+new_in_version: 3.13.3
+---
+
+This page guides you to connect {{% ttig %}} to {{% tts %}}.
+
+<!--more-->
+
+{{% ttig %}} is an 8 Channel LoRaWAN gateway, whose technical specifications can be found in [the official documentation](https://www.thethingsnetwork.org/docs/gateways/thethingsindoor/). 
+
+{{< figure src="TTIG.jpeg" alt="{{% ttig %}}" >}}
+
+## Prerequisites
+
+1. User account on {{% tts %}} with rights to create Gateways and API Keys.
+2. The EUI of the gateway. This EUI is a **16 character** value that can be found at the bottom in the WiFi setup screen.
```

Comment:
At this point you haven't connected to your gateway to WiFi yet. Perhaps just derive it from the MAC on the sticker?

So start with the `58A0CB` on the sticker, then `FFFE` if it's not on the sticker, then `80****`.